.github/workflows/x-keycloak-operator.yml

```diff
@@ -146,3 +146,87 @@ jobs:
+        run: echo ::set-output name=previous_tag::$(git describe --tags $(git rev-list --tags --max-count=1))
+
+      - name: Create OLM Bundle
+        if: ${{ inputs.tag != 'nightly' }}
+        working-directory: keycloak
+        run: ./operator/scripts/create-olm-bundle.sh ${{ inputs.tag }} ${{ steps.vars.outputs.previous_tag }} quay.io/${{ inputs.quay-org }}/keycloak-operator
+
+      - name: Clone community-operators
+        if: ${{ inputs.tag != 'nightly' }}
+        uses: actions/checkout@v3
+        with:
+          # keycloak-rel/community-operators
+          repository: keycloak-rel-testing/community-operators
+          path: community-operators
+          token: ${{ secrets.GH_TOKEN }}
+
+      - name: Automatic Community PR opening
+        if: ${{ inputs.tag != 'nightly' }}
+        working-directory: community-operators
+        # git remote add upstream https://github.com/k8s-operatorhub/community-operators.git
+        run: |
+          export UPSTREAM_REPO=https://github.com/mabartos/community-operators.git
+          
+          git remote add upstream $UPSTREAM_REPO
+          git fetch upstream
+          git reset --hard upstream/main
+          
+          mkdir -p operators/keycloak-operator/${{steps.version.outputs.version}}
+          cp -r ../operator/olm/${{steps.version.outputs.version}}/* operators/keycloak-operator/${{steps.version.outputs.version}}/
+          
+          git config --global user.email "keycloak-bot@keycloak.com"
+          git config --global user.name "keycloak-bot"
+          
+          git add .
+          git commit -s -m "Bump Keycloak operator to ${{steps.version.outputs.version}}"
+          
+          git tag -f ${{ inputs.tag }}
+          git push origin ${{ inputs.tag }} --force
+          
+          gh pr create --title "Bump Keycloak operator to ${{steps.version.outputs.version}}" --fill --repo $UPSTREAM_REPO
+          
+          echo ::set-output name=last_commit_rev::$(git rev-parse HEAD)
+
+      - name: Clone community-operators-prod
+        if: ${{ inputs.tag != 'nightly' }}
+        uses: actions/checkout@v3
+        with:
+          # keycloak-rel/community-operators-prod
+          repository: keycloak-rel-testing/community-operators-prod
+          path: community-operators-prod
+          token: ${{ secrets.GH_TOKEN }}
+
+      - name: Automatic Production PR opening
+        if: ${{ inputs.tag != 'nightly' }}
+        working-directory: community-operators-prod
+        # export DOWNSTREAM_REPO=https://github.com/redhat-openshift-ecosystem/community-operators-prod.git
+        run: |
+          export DOWNSTREAM_REPO=https://github.com/mabartos/community-operators-prod.git
+          
+          git remote add downstream $DOWNSTREAM_REPO
+          git fetch downstream
+          git reset --hard downstream/main
+          
+          git cherry-pick ${{ steps.vars.outputs.last_commit_rev }}
```

Comment:
Not sure cherry-pick is the right approach here. Could that result in conflicts? Would it be better to just apply the same stuff again?

Reply:
I'm not aware of any conflicts which would cause that. But I've changed that.